routes/web.php

```diff
@@ -20,9 +20,18 @@
 Route::middleware(['auth', 'verified'])->group(function () {
     Route::get('/expanded-reminders', 'ExpandedReminderListController@index');
 
-    Route::post('/reminders', 'RemindersController@store');
-    Route::patch('/reminders/{reminder}', 'RemindersController@update');
-    Route::get('/reminders', 'RemindersController@index');
+    /*
+    |--------------------------------------------------------------------------
+    | Families Routes
+    |--------------------------------------------------------------------------
+    |
+    */
+    Route::prefix('reminders')->name('reminders.')->group(function () {
```

Comment:
We rather keep routes explicit, since there aren't that many to begin with.
Also, this is out of scope of the PR.